steps/property-inspection/pets.tsx

```diff
@@ -75,51 +75,73 @@ const step = {
             ) as React.ReactNode,
             checkboxes: [
               {
-                label: "Amphibian (e.g. frog, newt, salamander, toad)",
+                label: {
+                  children: "Amphibian (e.g. frog, newt, salamander, toad)"
+                },
```

Comment:
Again, we should be able to insulate from this by making our `Checkboxes` map `label: string` to `label.children`. Given that the tests pass, it's not **necessary**, though.